docs/docs/cmd/planner/bucket/bucket-remove.md

```diff
@@ -0,0 +1,57 @@
+# planner bucket remove
+
+Removes the Microsoft Planner bucket from a plan
+
+## Usage
+
+```sh
+m365 planner bucket remove [options]
+```
+
+## Options
+
+`-i, --bucketId [bucketId]`
+: ID of the bucket to remove. Specify either `bucketId` or `bucketName` but not both.
+
+`-n, --bucketName [bucketName]`
+: Name of the bucket to remove. Specify either `bucketId` or `bucketName` but not both.
+
+`--planId [planId]`
+: ID of the plan to which the bucket to remove belongs. Specify either `planId` or `planName` when using `bucketName`.
+
+`--planName [planName]`
+: Name of the plan to which the bucket to remove belongs. Specify either `planId` or `planName` but not both.
+
+`--ownerGroupId [ownerGroupId]`
+: ID of the group to which the plan belongs. Specify `ownerGroupId` or `ownerGroupName` when using `planName`.
+
+`--ownerGroupName [ownerGroupName]`
+: Name of the group to which the plan belongs. Specify `ownerGroupId` or `ownerGroupName` when using `planName`.
+
+`--confirm`
+: Confirm removal of bucket.
+
```

Comment:
Let's add a reference to global options here. See https://github.com/pnp/cli-microsoft365/blob/e2f87ddd1cd1f30ae3a29e136be1d3366c8693a2/docs/docs/cmd/aad/app/app-add.md?plain=1#L58